contracts/modules/Permit2Payments.sol

```diff
@@ -23,7 +23,7 @@ abstract contract Permit2Payments is Payments {
 
     /// @notice Performs a batch transferFrom on Permit2
     /// @param batchDetails An array detailing each of the transfers that should occur
-    function permit2TransferFrom(IAllowanceTransfer.AllowanceTransferDetails[] memory batchDetails, address owner)
+    function permit2TransferFrom(IAllowanceTransfer.AllowanceTransferDetails[] calldata batchDetails, address owner)
```

Comment:
i noticed that this could also be changed to calldata
https://github.com/Uniswap/universal-router/blob/c853d4eef1f394c922f764511db6c6c8e4c3f8f2/contracts/modules/uniswap/v2/UniswapV2Library.sol#L125